main.go

```diff
@@ -42,6 +43,7 @@ type config struct {
 	ExportMethod                    string `env:"export_method,opt[none,app-store,development,developer-id]"`
 	CustomExportOptionsPlistContent string `env:"custom_export_options_plist_content"`
 
+	XcodebuildOptions         string `env:"xcodebuild_options"`
```

Comment:
Please have the same key and description as in xcode-archive.

Reply:
It is the same